07LessonSeven/homework/index.html

```diff
@@ -1,17 +1,53 @@
 <!DOCTYPE html>
-<html>
+<html class="clearfix">
 
   <head>
-    <title>NASA</title>
-    <link rel="stylesheet" href="css/style.css">
+    <title>Professional Portal</title>
+    <link rel="stylesheet" href="css/style.css" />
   </head>
 
-  <section id="imaginethat">
-<img =src"/>
-</img>
-  </section>
   <body>
+    <!--header color should be white until nav  -->
+    <header>
+      <div>
+        <h1>Professional Portal</h1>
+      </div>
+
+      <!--add background color in css for nav  -->
+      <nav class="nav_menu" class="clearfix">
```

Comment:
Thank you! 
Going to go back through all of my work and clean this sort of stuff up. Note to self: I can have multiple classes but they are contained within one set of quotes.